packages/x-date-pickers-pro/src/StaticNextDateRangePicker/StaticNextDateRangePicker.tsx

```diff
@@ -37,7 +36,7 @@ const StaticNextDateRangePicker = React.forwardRef(function StaticNextDateRangeP
     showToolbar: defaultizedProps.showToolbar ?? displayStaticWrapperAs === 'mobile',
   };
 
-  const { renderPicker } = useStaticPicker<DateRange<TDate>, TDate, 'day', typeof props>({
+  const { renderPicker } = useStaticRangePicker<TDate, 'day', typeof props>({
```

Comment:
We no longer need the `TValue` generic since we have one hook for single item editing and one hook for range editing